.circleci/config.yml

```diff
@@ -1,7 +1,319 @@
+              tp2t795UlrvaLgFI4wITqJF3LoTfy2MZu8JYCzlKM5pZksmEmJfR0RDAot2grtD3
+              H5A+PZfUIZ/8BhmdaOAv5i647unfVF6UpPYejZ0rb67oEazxdeIHK3aD5AjurdsO
+              UpW/PMwsbaltp4iI7hvUfRX7Afb5fPXIhv9pHh1xWYl3djUNWmFoiMMP4tuxpOBo
+              y+T4maQaiDSHAoIBADrlZ9EIMclMnNXJYE4O4fbFesUvV0lHM3+ayQgXiH0Vg5Nl
+              2xjPlqBX0bDajVluPU6AF3GYxfoSLv1GXqTvb9iVpKXrAHp+nef0uxMP9ltZT6Qz
+              UA1wh3x2OBFJ0hK0B1FsmeSHS8VDQye615jEA8iMM/GrbnnM/p7ccEcOkyO8YJSj
+              I/rNbzN6u8yAPZCzyx6Hy4w/xsdf1acslOHJj3kyX/cwqCGxnc/GvVR2OSZyHVnT
+              sLnGj7NEeudwvKlyxuzj5CMmz111wVEI2olgQa9Sl+EBu140mnDNTNYCA7OnwE3z
+              GoFMOrXC2mf2ZfSge4orbL5Nellnt51pOLp2x8ECggEBALM8Mazw/FOF9mbdgjJM
+              PFGSaa7rBcVJwdHttDHBmlPI6wzsvFEMPru6nfx76KJQbORqK9r13sN5fyzof59m
+              TwsbMt/cFSnOQJ39M7YPstDofbl20cDOduUzpEVsRvVKokhqGB3XVRiuZ1y+8WSz
+              Wh7OiTu3AwzKsrcYXkZQdnlRBq0iYcfLPKzHqUJLLzbOH9Q6djL5c8V/qLNfvNI1
+              2HqKVqV8Ex+zKJhBWRAe+x3bKnbS7MPQ6zNfsOdgCmhydwRCquPzpr7JU/PFZh+4
+              b31cHgFrIZR2d2AzW1XcSLzsqa2vUs2RKOIu2deAPaUI/66zCZeTnGBNEFza76Ga
+              1oUCggEAA38oXcnputwL103SeD8+uwHjtTf183Rucr+Ryqz6GymiWjlzELqu7TRd
+              yadAaNg9CuXmYS33Jtk/UNS0k9FvYqGTR+SBXIZr6nt9ZFd0SNlQkwkAQCsuekEs
+              nJlmUZax7DxXMgIHMKDboHZYM/MhgzEGSALmhU5LZ76MS17v3NEPxYpVHxjAotxW
+              g03HjWTltS8Bgt6u0KFTGJKEUcfwvWKZtjk5Fc1heZ49zh1nU3zo9C/h8iiijTy2
+              s/YksP6cxveae4b7soN4rD/vnfsmKcG+DnTf6B8Zbm6tI2TneYOfFSCryp+yDnaJ
+              PIDNiTxNecePOmrD+1ivAEXcoL+e1w==
+              -----END PRIVATE KEY-----
+
+        - id: webextensions-rsa-with-recommendation
+          type: xpi
+          # The signing parameters for each type of add-on are 'add-on' are
+          # signed with the OU 'Production' and the provided ID
+          # 'add-on-with-recommendation' are signed with the OU 'Production'
+          # and the provided ID and add a recommendation file 'extension' are
+          # signed with the OU 'Mozilla Extensions' and the provided ID 'system
+          # add-on' are signed with the OU 'Mozilla Components' and the
+          # provided ID 'hotfix' are signed with the OU 'Production' and the ID
+          # 'firefox-hotfix@mozilla.org'
+          mode: add-on-with-recommendation
+          recommendation:
+            path: "mozilla-recommendation.json"
+            states:
+              recommended: true
+              recommended-android: true
+              verified: true
+              line: true
+            relative_start: 0h
+            duration: 26298h
+          # RSA key gen is slow and CPU intensive, so we can optionally
+          # pregenerate and cache keys with a worker pool
+          rsacacheconfig:
+            numkeys: 25
+            numgenerators: 2
+            generatorsleepduration: 1m
+            fetchtimeout: 100ms
+            statssamplerate: 1m
+          certificate: |
```

Comment:
Would there be a way to reference these as external files?

Reply:
no